packages/EasyApiToken/tests/AbstractAuth0JwtTokenTestCase.php

```diff
@@ -45,16 +45,14 @@ protected function createAuth0JwtDriver(
         ?string $audienceForEncode = null,
         ?array $allowedAlgos = null
     ): JwtDriverInterface {
-        $auth0Driver = new Auth0JwtDriver(
+        return new Auth0JwtDriver(
             $validAudiences ?? static::$validAudiences,
             $authorizedIss ?? static::$authorizedIss,
-            'localhost',
+            'example.com',
```

Comment:
We need this after this excellent commit https://github.com/auth0/auth0-PHP/commit/e89127a97ad72715e1a55c57fc3455146e12d9a8